### PR TITLE
Migrate Card story to typescript

### PR DESCRIPTION
**Pull request by wenche**
Part of #735

### CODE REVIEW

**Comment by wenche** [comment]
I'm having an issue with the first tab being renamed to Story. Will hopefully be [fixed soon](url)https://github.com/storybookjs/storybook/issues/12357

**Comment by wenche** [comment]
I had to use the `JSX.IntrinsicElements` trick for the args table to show. I wasn't enough with `React.HTMLAttributes` 🤷‍♀️

File: libraries/core-react/src/Card/CardActions.tsx [hunk 1]
```diff
@@ -16,7 +18,7 @@ const StyledCardActions = styled.div<React.CSSProperties>`
   justify-content: ${({ justifyContent }) => justifyContent};
 `
 
-export const CardActions = forwardRef<HTMLDivElement, Props>(
+export const CardActions = forwardRef<HTMLDivElement, CardActionProps>(
   function EdsCardActions(
```
Comment (mimarz) [inline libraries/core-react/src/Card/CardActions.tsx:22]: Missed function renameing

File: apps/storybook-react/stories/Card.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,347 @@
+import React from 'react'
+// import { withKnobs, select, boolean } from '@storybook/addon-knobs'
```
Comment (mimarz) [inline apps/storybook-react/stories/Card.stories.tsx:2]: Suggest we just remove this if we aren't using. Going to be replace by `controls` anyways #757

**Review by mimarz — CHANGES_REQUESTED** [review]
Some small stuff

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Pull request merged**